spark-icons/lint.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+    <issue id="VectorPath" severity="informational" />
```

Comment:
We should probably filter by path

```suggestion
    <issue id="VectorPath" path="spark-icons/src/main/res-iconography/**/*.xml" severity="informational" />
```

Reply:
And even remove it to avoid triggering the Code scanning thing?

Reply:
I didn't know we could do this 😮

Reply:
And add an xml comment

```xml
<!-- ... -->
```